docs/source/peer_ledger_snapshot.md

```diff
@@ -5,24 +5,26 @@ For a peer to process transactions on a channel, it must contain the minimum led
+* While endorsements will continue and queries can be submitted, peers taking a snapshot at a particular height will not commit blocks on the channel while the snapshot is being generated. Because taking a snapshot is a resource-intensive operation, the peer might also be slow to endorse transactions or commit blocks on other channels. For these reasons, it is anticipated that snapshots will only be taken when necessary (for example, when a new peer needs a snapshot to join a channel, or when organizations want to verify that no ledger forks have occurred).
+* Because the private data between organizations in a channel is likely to be at least somewhat different, private data is not included in the snapshot (hashes of the private data are included, but not the data itself). Peers that join a channel using a snapshot will discover the collections it is a member of and pull the relevant private data from peers that are members of those collections directly. This private data reconciliation will begin after the peer joins the channel, and may take some time.
 * The snapshot process does not archive and prune the ledgers of peers that are already joined to the channel. Similarly, it is not meant as a method to take a full backup of a peer, as private data, and peer configuration information, such as MSPs, are not included in a snapshot).
 * It is not possible to use the `reset`, `rollback`, or `rebuild-dbs` commands on peers that have joined a channel using a snapshot since the peer would not have all the block files required for the operations. Instead of these administrative commands, it is expected that peers that have joined a channel using a snapshot can be entirely rebuilt from the same or newer snapshots.
 
-Because of these limitations, the decision of whether a peer will join a channel from a snapshot depends on a few factors.
+## Considerations
 
-* **The current ledger height of the channel**. The smaller the ledger, the less useful it is to join a channel using a snapshot. Where the size of the ledger crosses a point where it makes sense to use a snapshot depends on your use case.
-* **The importance of querying old data**. Some channels and use cases might call for querying old blocks regularly, while in other channels it might only be necessary to know the current state. This must be balanced by the server costs experienced by your organization.
+* When deciding whether to join peers from a genesis block or from a snapshot, consider the time it may take to join a peer based on the number of blocks since the genesis block, whether the peer will be able to pull blocks from the ordering service based on the original channel configuration in the genesis block, and whether you will need to query the history of channel since the genesis block (historical blocks, transactions, or state).
+* If your endorsement requests or queries don't require the latest block commits, you can target a peer that generates snapshots.
```

Comment:
to join a peer to a channel

Feels like a lot of "genesis blocks" here too. Maybe the last phrase could be:

and whether you will need to query the entire history of a channel (historical blocks, transactions, or state).